### PR TITLE
adds wandb to CI, fixes seed

### PR DESCRIPTION
**Pull request by charlesfrye**
see [this dashboard](https://wandb.ai/cfrye59/ci-llm-finetuning-overfit-sqlqa/reports/LLM-Fine-Tuning-CI--Vmlldzo4MDc0Njk0)

### CODE REVIEW

**Comment by charlesfrye** [comment]
`[SQL]` is sometimes getting left off -- should separate out a validation set to reduce randomness so i don't have to go fishing for a seed that gives good behavior.
